demos/dotnet/dotnet/Helpers.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+namespace dotnet
+{
+    internal static class Helpers
```

Comment:
These are System.IO.File extensions? FileSystemHelpers? Helpers seems like a dumping-ground sort of name...